data/json/items/gun/50.json

```
@@ -94,7 +109,15 @@
       "barrel_length": -1
     },
     "flags": [ "RELOAD_EJECT" ],
-    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "50": 1 } } ]
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
```

```suggestion
        "pocket_type": "MAGAZINE",
```
Related to the other comment